hack/local.sh

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env bash
+
+set -eu
+set -o pipefail
+
+cd $(dirname "${BASH_SOURCE[0]}")/..
+
+function usage() {
+  cat <<-USAGE
+local.sh [OPTIONS]
+
+Builds and generates a deployment yaml for kpack.
+
+Prerequisites:
+- pack or ko installed
+- kubectl installed
+- docker login to your registry
+
+OPTIONS
+  --help                          -h  prints the command usage
+  --buildType <buildType>             build system to use. valid options are pack or ko.
```

Comment:
I think CLI flags usually uses `kebab-case`

Reply:
ya I can change, I took the loop from paketo which uses camel case